schemas/erc20/schema.json

```diff
@@ -0,0 +1,57 @@
+{
+  "schema": "ERC-20",
+  "description": "Table definitions for the ERC-20 schema. These tables can be used for any protocol.",
+  "protocolCategory": ["ERC20"],
+  "version": "1.0.0-alpha",
+  "tables": [
+    {
+      "label": "Token ERC-20 Transfers",
+      "tableName": "general_token_erc20_transfers",
+      "aggregation": "transaction",
+      "uniqueKey": ["transaction_hash", "log_index"],
+      "description": "User level token balance snapshots.",
```

Comment:
nit: this should be saying something about erc20 trasfer events

Reply:
sorry, missed that part